tests/js/spec/views/stream/searchBar.spec.jsx

```diff
@@ -275,7 +275,7 @@ describe('SearchBar', function() {
         pinnedSearch: {id: '1', query: 'url:"fu" '},
       };
       const searchBar = mount(<SearchBar {...props} />, routerContext);
-      searchBar.find('PinButton').simulate('click');
+      searchBar.find('button[data-test-id="pin-search"]').simulate('click');
```

Comment:
We should consider requiring aria-labels for all Buttons for accessibility and use it as selectors for tests

Reply:
I can add aria labels to these buttons.